render-superhero.js

```diff
@@ -0,0 +1,19 @@
+
+export function renderSuperhero(heroData) {
+    const div = document.createElement('div');
+    const heroProfile = document.createElement('p');
+    const alterEgoEl = document.createElement('p');
+
+    heroProfile.classList.add('hero');
+    alterEgoEl.classList.add('alter-ego');
+
+    
+
+    heroProfile.textContent = heroData.name;
+    alterEgoEl.textContent = `Alter Ego: ${heroData.alterEgo}`;
```

Comment:
what is the shape of the data of heroData.alterEgo? Try going just a little deeper into that object.